crates/pop-parachains/src/build.rs

```diff
@@ -1,30 +1,68 @@
 // SPDX-License-Identifier: GPL-3.0
 
-use crate::errors::Error;
+use crate::Error;
+use anyhow::Result;
 use duct::cmd;
-use std::path::Path;
+use pop_common::{parse_package_name, replace_in_file};
+use serde_json::Value;
+use std::{
+	collections::HashMap,
+	fs,
+	path::{Path, PathBuf},
+};
 
-/// Build the parachain located in the specified `path`.
+/// Enum representing the build profile for a parachain.
+#[derive(Debug, PartialEq)]
+pub enum Profile {
+	/// Debug profile, optimized for debugging.
+	Debug,
+	/// Release profile, optimized without any debugging functionality.
+	Release,
+}
+
+impl Profile {
+	/// Returns the corresponding path to the target folder.
+	fn target_folder(&self, path: &Path) -> PathBuf {
+		match self {
+			Profile::Release => path.join("target/release"),
+			Profile::Debug => path.join("target/debug"),
```

Comment:
Keep in mind that adding `Other(String)` will result in `Profile::Other(profile) => path.join("target").join(profile)`

Reply:
Same as above, I prefer to add new profiles once needed.